src/crafting.cpp

```diff
@@ -404,8 +404,10 @@ bool player::check_eligible_containers_for_crafting( const recipe &rec, int batc
         }
 
         if( charges_to_store > 0 ) {
-            popup( _( "You don't have anything to store %s in!" ), prod.tname() );
-            return false;
+            if( !query_yn( ( "You don't have anything in which to store %s and may have to pour it out or consume it as soon as it is prepared! Proceed?" ),
```

Comment:
Needs to be translatable - see "doc/TRANSLATING.md".

Reply:
ok I think I know what you mean.